.github/workflows/python.yml

```diff
@@ -0,0 +1,47 @@
+name: Python
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.12']
+    runs-on: ${{ matrix.os }}
+    env:
+      working-directory: ./bindings/python
+
+    defaults:
+      run:
+        working-directory: ${{ env.working-directory }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: install rust stable
```

Comment:
Nitpick: use start sentences and names with capitals. `Install Rust (stable)`. Similarly, fix "limbo" to "Limbo" in a step name below.